plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -509,6 +512,37 @@ class GesturesPluginTest {
     verify { cameraAnimationsPlugin.easeTo(any(), any()) }
   }
 
+  @Test
+  fun verifyScaleWithLowLevelAnimation() {
```

Comment:
let's rename test to identify that in this case `simultaneousRotateAndPinchToZoomEnabled = false`